public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -241,6 +240,11 @@ function Main (params) {
                 });
             }
         });
+        $("#feedback-button-tooltip").tooltip({
```

Comment:
Why doesn't the feedback tooltip work like the other ones, and needs it's own call here? Why can't it be included in the `$('[data-toggle="tooltip"]').tooltip({` bit below?